components/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC, useState, useEffect } from 'react';
+import Sidebar from './SideBar';
+import NavBar from './NavBar';
+
+const Modal: FC = () => {
+  const [show, setShow] = useState<boolean>(true);
```

Comment:
Since it's no longer for testing.

```suggestion
  const [show, setShow] = useState<boolean>(false);
```